templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h1>Do you really want to delete this manufacturer {{ manufacturer.name }} from {{ manufacturer.country }}?</h1>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete please" class="btn btn-danger" />

```

Comment:
Add a `Cancel` button that will lead to the previous page as mentioned in the [checklist](https://github.com/mate-academy/py-taxi-service-forms/blob/master/checklist.md).